.travis.yml

```diff
@@ -9,11 +9,18 @@ before_install:
   - sudo curl https://github.com/mikefarah/yq/releases/download/3.2.1/yq_linux_amd64 -L -o /usr/local/bin/yq-3.2.1
   - sudo chmod +x /usr/local/bin/yq-3.2.1
   - sudo ln -s /usr/local/bin/yq-3.2.1 /usr/local/bin/yq
-  - sudo apt install shellcheck
-script: bash ci/build.sh
+  - wget -qO- https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz | tar -xJf -
+  - sudo cp shellcheck-v0.7.1/shellcheck /usr/local/bin
+  - rm -rf shellcheck-v0.7.1/
```

Comment:
I believe the only thing I could suggest to rethink is the usage of `curl` in one place and `wget` elsewhere.

I'd personally stick to `curl` wherever possible.